setup.py

```diff
@@ -218,9 +218,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'cassandra-driver>=3.13.0,<4',
 ]
 celery = [
-    'celery~=4.4.2',
-    'flower>=0.7.3, <1.0',
-    'vine~=1.3',  # https://stackoverflow.com/questions/32757259/celery-no-module-named-five
+    'celery~=5.1.2',
```

Comment:
This is equivalent to `>=5.1.2,<5.2` which probably isn't what we intend

```suggestion
    'celery~=5.1,>=5.1.2',
```